easybuild/easyblocks/m/maple.py

```diff
@@ -55,39 +55,37 @@ def install_step(self):
         else:
             raise EasyBuildError("Could not locate installer in %s", self.builddir)
 
-        qa = {
-            'PRESS <ENTER> TO CONTINUE:': '',
-            "Press [Enter] to continue:": '',
-            'DO YOU ACCEPT THE TERMS OF THIS LICENSE AGREEMENT? (Y/N):': 'Y',
-            "Do you accept this license? [y/n]:": 'y',
-            'ENTER AN ABSOLUTE PATH, OR PRESS <ENTER> TO ACCEPT THE DEFAULT :': self.installdir,
-            'IS THIS CORRECT? (Y/N):': 'Y',
-            'Language Selection\n\nPlease select the installation language\n[1] English - English\n[2] Japanese - \n'
-            'Please choose an option [1] : ': '1',
-            'Do you wish to have a shortcut installed on your desktop? ->1- Yes 2- No ENTER THE NUMBER ' +
-            'FOR YOUR CHOICE, OR PRESS <ENTER> TO ACCEPT THE DEFAULT::': '2',
-            "Do you wish to have a shortcut installed on your desktop? [Y/n]:": 'n',
-            '->1- Single User License 2- Network License ENTER THE NUMBER FOR YOUR CHOICE, ' +
-            'OR PRESS <ENTER> TO ACCEPT THE DEFAULT::': '2',
-            'PRESS <ENTER> TO EXIT THE INSTALLER:': '',
-            'License server (DEFAULT: ):': self.cfg['license_server'],
-            "License server []:": self.cfg['license_server'],
-            'Port number (optional) (DEFAULT: ):': self.cfg['license_server_port'] or '',
-            '->1- Configure toolbox for Matlab 2- Do not configure at this time ENTER THE NUMBER FOR YOUR CHOICE, ' +
-            'OR PRESS <ENTER> TO ACCEPT THE DEFAULT::': '2',
-            "MATLAB Configuration [y/N]:": 'n',
-            "Check for updates now [Y/n]:": 'n',
-            "Use proxy server when checking for updates [y/N]:": 'n',
-            "Downloads & Service Packs. [Y/n]:": 'n',
-        }
-        std_qa = {
-            r"Choose Install Folder \[.*\]:": self.installdir,
-            r"\[2\] Network License.*\nPlease choose an option \[.\] :": '2',
-            r"\[1\] Single Server.*\n.*\nPlease choose an option \[.\] :": '1',
-            r"Port number \[[0-9]+\]:": self.cfg['license_server_port'] or '',
-            r"Enable periodic checking for Maple .* updates after installation \[Y/n\]:": 'n',
-            r'Pre-Installation Summary[\s\S]*': '',
-        }
+        qa = [
+            (r'PRESS <ENTER> TO CONTINUE:', ''),
+            (r"Press \[Enter\] to continue:", ''),
+            (r'DO YOU ACCEPT THE TERMS OF THIS LICENSE AGREEMENT? (Y/N):', 'Y'),
+            (r"Do you accept this license? [y/n]:", 'y'),
+            (r'ENTER AN ABSOLUTE PATH, OR PRESS <ENTER> TO ACCEPT THE DEFAULT :', self.installdir),
+            (r'IS THIS CORRECT? \(Y/N\):', 'Y'),
+            (r'Language Selection\n\nPlease select the installation language\n\[1\] English - English\n'
+             r'\[2\] Japanese - \nPlease choose an option \[1\] : ', '1'),
```

Comment:
This one is failing, there are some special characters in here, so loosening this up a bit...

```suggestion
            (r'Language Selection\n\nPlease select the installation language\n\[1\] English - English\n'
             r'\[2\] Japanese.*\nPlease choose an option \[1\] : ', '1'),
```